src/BigBlueButton.php

```diff
@@ -61,21 +62,22 @@ class BigBlueButton
     protected $jSessionId;
 
     /**
-     * @param  string     $baseUrl (optional)
-     * @param  string     $secret  (optional)
+     * @param string $baseUrl (optional)
+     * @param string $secret  (optional)
```

Comment:
I think the format is enforced by cs-fixer, but it could also occur because we are using different versions through the fact that the composer.lock is ignored and not committed.

Reply:
In my opinion, with the new version php requirements we should use type hinting instead of PHPDoc.

Reply:
@Yurujai I think we should use both, whenever possible.

All those formatting changes are really annoying during code review. Hopefully @sualko PR will solve this.